### DISCUSSION TITLE
Method to update prompt template for Langserve use case

### DISCUSSION THREAD

**Discussion by quadcube** [langfuse/langfuse#2498] [Support] [answered]
The method specified in the doc works fine when we execute the chain from the initialization of the prompt to the invoke. 
https://langfuse.com/docs/prompts/example-langchain

However, for Langserve, we pre-initialised the prompt_template, but if there's a new version on the Langfuse UI, the prompt_template does not update due to how it was initialised. Any ideas of how to get it pickup the latest changes when it is invoked?

Example code as below:
```
langfuse = Langfuse()
langfuse_handler = CallbackHandler()
config = RunnableConfig(callbacks=[langfuse_handler])

chat_model = ChatOpenAI()

prompt_langfuse = langfuse.get_prompt("prompt_name", cache_ttl_seconds=0)
prompt_template = ChatPromptTemplate.from_messages(prompt_langfuse.get_langchain_prompt())

chain = prompt_template | chat_model | StrOutputParser()
add_routes(
    app,
    chain.with_types(input_type=InputChat).with_config(config),
    path="/some_chain",
)
```

**Comment by JayAhn2** [comment]
I'm stuck with the exactly same issue. 

I found that you can wrap the chain with RunnableLambda, but it wouldn't create the openapi doc correctly.

**Reply by quadcube** [reply, marked as answer]
I solved it in this manner for regular LCEL chain.

```
def get_prompt(input):
    prompt_langfuse = langfuse.get_prompt(prompt_name, type="chat", label="production")
    prompt_template = ChatPromptTemplate.from_messages(prompt_langfuse.get_langchain_prompt())
    return prompt_template

chain = get_prompt | chat_model | StrOutputParser()
```
